CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+## master / unreleased
+
+## 0.1.0-rc.0 / 2019-07-31
+
+* [CHANGE] HA Tracker flags were renamed to provide more clarity #1465
```

Comment:
Maybe explicitly mark this as Breaking Change?

Reply:
The goal is for [CHANGE] to indicate that something is breaking. I followed how Prometheus does annotations, for example config being removed or metrics being renamed recently: https://github.com/prometheus/prometheus/blob/master/CHANGELOG.md

I guess there are different levels of breaking (metrics changing, vs something potentially not starting), but I think it could be hard to differentiate levels of changes.